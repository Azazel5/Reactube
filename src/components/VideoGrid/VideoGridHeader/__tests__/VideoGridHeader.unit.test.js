import React from 'react'
import {shallow} from 'enzyme'
import VideoGridHeader from '../VideoGridHeader'

describe('<VideoGridHeader />', () => {
    test('renders <VideoGridHeader /> without props', () => {
        const wrapper = <VideoGridHeader />
        expect(wrapper).toMatchSnapshot()
    })

    test('renders <VideoGridHeader /> with empty title', () => {
        const wrapper = <VideoGridHeader title=""/>
        expect(wrapper).toMatchSnapshot()
    })

    test('renders <VideoGridHeader /> with title', () => {
        const wrapper = <VideoGridHeader title="A title"/>
        expect(wrapper).toMatchSnapshot()
    })
})