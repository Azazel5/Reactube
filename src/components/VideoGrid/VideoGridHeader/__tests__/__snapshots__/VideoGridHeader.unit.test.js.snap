// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<VideoGridHeader /> renders <VideoGridHeader /> with empty title 1`] = `
<videoGridHeader
  title=""
/>
`;

exports[`<VideoGridHeader /> renders <VideoGridHeader /> with title 1`] = `
<videoGridHeader
  title="A title"
/>
`;

exports[`<VideoGridHeader /> renders <VideoGridHeader /> without props 1`] = `<videoGridHeader />`;
